CMS/templates/contentPages/asset_type_page.html

```diff
@@ -34,20 +34,22 @@ <h1>{{page.title}}</h1>
                     </div>
                     <div id="search-results" class="spacer-top-1">
                         <div class="row">
-                            <ul class="crc-thumbnails">
```

Comment:
Does this card need this outer level of `<ul>` and `<li>` to list out the lists? It is never going to have more than one item. If it doesn't then just changing the `<ul>` to a `<div>` would sort the problem at this level

The bottom section is fine but it looks to me our problem at this level that this `<ul>` should never have been a `<ul>` to start off with

Note: The big deal would be for screen readers - extra structure gets announced by screen readers so we are currently reading out `unordered list, list item one, unordered list, list item one, Posters`